### PR TITLE
Improve `docker image rm` reference docs

### PR DESCRIPTION
**Pull request by thaJeztah**
Copies the improved description from the man page (made in https://github.com/docker/cli/pull/1444/) to the online reference docs.

/cc @SvenDowideit 👍

### CODE REVIEW

**Comment by thaJeztah** [comment]
ping @vdemeester @silvin-lubecki PTAL 🙏

File: docs/reference/commandline/rmi.md [hunk 1]
```diff
@@ -26,6 +26,17 @@ Options:
...
+## Description
+
+Removes (and un-tags) one or more images from the host node. If an image has
+multiple tags, using this command with the tag as a parameter only removes the
+tag. If the tag is the only one for the image, both the image and the tag are
+removed.
+
+This does not remove images from a registry. You cannot remove an image of a
+running container unless you use the `-f` option. To see all images on a host
+use the [`docker image ls`](images.md) command.
```
Comment (silvin-lubecki) [inline docs/reference/commandline/rmi.md]: Can you add a small paragraph about the `--no-prune` flag too? I think its behavior is not that obvious regarding its name and then deserve some explanation 😄 .

Comment (thaJeztah) [reply docs/reference/commandline/rmi.md]: @silvin-lubecki there's definitely more improvements to be made to this page; this PR was only intended to port the changes made in https://github.com/docker/cli/pull/1444 to the online documentation, so I'd rather keep that for a separate change (I'll see if I can make some time for additional improvements).

For that follow-up; if we _do_ add that description, it should likely go into the `examples` section.

**Comment by aidan-fitz** [comment]
@thaJeztah Thanks for creating this pull request! I was trying to submit an issue about this myself.

**Comment by aidan-fitz** [comment]
This change shows up at [docker rmi] but not [docker image rm].

[docker rmi]: https://docs.docker.com/engine/reference/commandline/rmi/
[docker image rm]: https://docs.docker.com/engine/reference/commandline/image_rm/

**Comment by thaJeztah** [comment]
That's being discussed; likely redirects will be made to redirect the pages to their canonical command

**Pull request merged**